docs/localdev.md

```diff
@@ -1,88 +1,135 @@
+## Working with the `ExternalDNS` Operator in CRC
+Prerequisites:
+- You have a running instance of CRC.
 
-This document describes howto get a local development enviroment setup, so that a developer can make code changes, fix bugs add new features, test and debug etc for the ExternalDNS operator \
-We will consider the following scenarios, working with CRC (code ready containers), Kind (kubernetes in Docker) and finally with any external Kubernetes/OpenShift cluster
+CRC is a pre-built Container Development Environment based on Red Hat Enterprise Linux which helps you develop container-based applications quickly. You can work with the `ExternalDNS` Operator in CRC using either of the following two methods:
+- Build and push to the internal CRC registry.
+- Run the `ExternalDNS` Operator locally.
 
-Please refer to these links for further info regarding 
-- CRC [link](https://developers.redhat.com/products/cdk/overview)
-- Kind [link](https://kind.sigs.k8s.io/docs/user/quick-start/)
-- External Kubernetes/OpenShift cluster
+### Building and pushing to the internal CRC registry
+1. Create a fork of the `openshift/external-dns-operator` repository in your GitHub account.
+2. Clone the `openshift/external-dns-operator` repository from your GitHub account to your local machine.
+3. Navigate to the downloaded `external-dns-operator` repository:
+    ``` bash
+    $ cd <projects-dir>/external-dns-operator
+    ```
+    `<projects-dir>` is the directory where you downloaded the  `external-dns-operator` repository.
+4. Verify that you can access your CRC environment:
+    ```bash
+    $ export KUBECONFIG=~/.crc/machines/crc/kubeconfig
+    ```
+5. Access the CRC internal registry as the `kubeadmin` user, using the podman tool:
+    ```bash
+    $ podman login -u kubeadmin -p $(oc whoami -t)    default-route-openshift-image-registry.apps-crc.testing --tls-verify=false
+    ```
+6. Follow the standard make recipes:
+- Create the `external-dns-operator` namespace:
+    ```bash
+    $ oc create ns external-dns-operator
+    ```
+- Set the `IMG` EnVar to the following URL:
+`default-route-openshift-image-registry.apps-crc.testing/<namespace>/<image-name>:tag`
+- Run the following commands to set the `IMG` EnVar:
+    ```bash
+    $ export IMG=default-route-openshift-image-registry.apps-crc.testing/external-dns-operator/external-dns-operator:dev
+    $ CONTAINER_ENGINE=podman make image-build
+    $ CONTAINER_ENGINE=podman TLS_VERIFY=false make image-push
+    ```
+- Verify that the image is pushed:
+    ```bash
+    $ oc -n external-dns-operator get is
+    ```
+7. Deploy an instance of ExternalDNS:
+- Deploy the `ExternalDNS` Operator:
+    ```bash
+    $ make deploy
+    ```
+- Switch to the `external-dns-operator`namespace:
+    ```bash
+    $ oc project external-dns-operator
+    ```
+- Verify that the `ExternalDNS` Operator is deployed:
+    ```bash
+    $ oc get all
+    ```
+- If you encounter any errors, run the following command:
```

Comment:
This is not another step. It should be part of the "Verify that the `ExternalDNS` Operator is deployed" step.